manage_breast_screening/mammograms/views/breast_cancer_history_views.py

```diff
@@ -134,7 +135,39 @@ def get_context_data(self, **kwargs):
             {
                 "heading": "Edit details of breast cancer",
                 "page_title": "Edit details of breast cancer",
+                "delete_link": {
+                    "text": "Delete this item",
+                    "class": "nhsuk-link app-link--warning",
+                    "href": reverse(
+                        "mammograms:delete_breast_cancer_history_item",
+                        kwargs={
+                            "pk": self.kwargs["pk"],
+                            "history_item_pk": self.kwargs["history_item_pk"],
+                        },
+                    ),
+                },
             },
         )
 
         return context
+
+
+class DeleteBreastCancerHistoryView(DeleteWithAuditView):
+    def get_thing_name(self, object):
+        return "item"
+
+    def get_success_message_content(self, object):
+        return "Deleted breast cancer"
+
+    def get_object(self):
+        provider = self.request.user.current_provider
```

Comment:
Minor point, but `request.current_provider` is also made available via middleware.